frontend/pulse3d/src/.env.production

```diff
@@ -1,2 +1,2 @@
-NEXT_PUBLIC_PULSE3D_URL=https://pulse3d.curibio.com 
-NEXT_PUBLIC_USERS_URL=https://apiv2.curibio.com/users
+NEXT_PUBLIC_PULSE3D_URL=https://pulse3d.curibio-test.com 
+NEXT_PUBLIC_USERS_URL=https://apiv2.curibio-test.com/users
```

Comment:
I was thinking we might want to create some new file, maybe `.env.testing` so this value doesn't get randomly changed. Or if this gets created automatically (I can't remember) maybe we just gitignore it

Reply:
yeah lets git ignore them.. there is some issues with exporting in next because the "test env" is considered when you run `npm run dev` so even though we're testing here, it's still considered production to nextjs.

Reply:
That's why we're dynamically changing them each time,  but we could probably find a way to use diff files in the future.